PWGMM/Tasks/uecharged.cxx

```diff
@@ -64,21 +65,11 @@ struct uecharged {
     globalTrackswoPrim.SetMaxDcaXYPtDep([](float pt) { return 3.f + pt; });
     globalTrackswoPrim.SetRequireGoldenChi2(false);
 
-    std::vector<double> ptBinningL = {
-      0.15, 0.50, 1.00, 1.50, 2.00, 2.50, 3.00, 3.50, 4.00, 4.50,
-      5.00, 6.00, 7.00, 8.00, 9.00, 10.0, 12.0, 14.0, 16.0, 18.0,
-      20.0, 25.0, 30.0, 40.0, 50.0};
-    AxisSpec ptAxisL = {ptBinningL, "#it{p}_{T}^{trig} (GeV/#it{c})"};
+    ConfigurableAxis ptBinningt{"ptBinningt", {0, 0.15, 0.50, 1.00, 1.50, 2.00, 2.50, 3.00, 3.50, 4.00, 4.50, 5.00, 6.00, 7.00, 8.00, 9.00, 10.0, 12.0, 14.0, 16.0, 18.0, 20.0, 25.0, 30.0, 40.0, 50.0}, "pTtrig bin limits"};
```

Comment:
Good, but it also requires the first entry in the list to be `VARIABLE_WIDTH` (which is 0), so that it can be correctly parsed by framework and hyperloop.

Reply:
Ok, but in this case the lowest edge of the first bin in 0.15. So, it should be ok. For ptAxis, I modified according to your suggestion. Thanks!

Reply:
Ok! It is just more readable with a semantic `VARIABLE_WIDTH`.